icubam/analytics/preprocessing.py

```diff
@@ -302,8 +303,14 @@ def spread_cum_jumps(d, icu_to_first_input_date):
       for col in cols:
         if col in already_fixed_col:
           continue
-        beg_val = dg.loc[dg.date == beg, col].values[0]
-        end_val = dg.loc[dg.date == end, col].values[0]
+        beg_val = dg.loc[dg.date == beg, col]
+        if not len(beg_val):
+          continue
```

Comment:
Otherwise this fails with key error on test data, particularly due to the hardcoded `date_begin_transfered_refused` above.